db.py

```diff
@@ -37,8 +37,16 @@ class Code(BaseModel):
 	created_at = DateTimeField(default=datetime.now())
 	used_at    = DateTimeField(null=True)
 
+	# This is all for automatic conversion to strings.
+	def __str__(self):
+		return self.code
+	def __add__(self, other):
+		return self.code + other
+	def __radd__(self, other):
+		return other + self.code
+
 	def get_by_code(code, include_used=False):
-		query = Code.select().where(Code.code == code)
+		query = Code.select().where(Code.code == str(code))
```

Comment:
I really like this because it's still backwards compatible with all the calls that pass in raw strings.